package.json

```diff
@@ -5,24 +5,25 @@
   "homepage": "https://sweetalert2.github.io/",
```

Comment:
you made changes in `package.json` please bump `yarn.lock` as well (`yarn install`), thanks!